added schema for data modeling blog (#23)
vendia/examples: design-patterns/domain-driven-design/pattern3/README.md

3. Fill out details in the Uni Creation Wizard. 

**Step 1:** Give your Uni a name. You can name it ‘test-CRMandMarketing-append-unique-suffix`. Time to get creative and choose a unique suffix that you will need for your Uni name. Reaplace it with the `append-unique-suffix` in the proposed Uni name. Click `Next`. 
  
**Step 2:** Fill out the following information to create nodes: Node 1 and Node 2. 

*Node 1* [ READ/WRITE on the `CustomerAccount`] and [READ Only on `CustomerProfile`]
  
```
Node Name: "CRM". // you can change the name.
Node Description: "Node dedicated Marketing". // You can change the description.
Cloud Server Provider: "AWS"
Node Region:"us-east-1"// you can change it
Auth Option: "Vendia User". // Let's use Vendia User
```

*Node 2* [ READ Only on the `CustomerAccount`] and [READ/WRITE on the `CustomerProfile`]

```
Node Name: "Marketing". // you can change the name.
Node Description: "Node dedicated to Marketing". // You can change the description.
Cloud Server Provider: "AWS"
Node Region:"us-east-1". //you can change it
Auth Option: "Vendia User". // Let's use Vendia User
```

> **Step 3:** Provide the Uni schema. Copy and paste the Uni schema from the schema.json file in the schema folder. The schema is the same as the one you used in Pattern 2. The difference is in the ‘aclnput’ you used in the GraphQL query that you used to add new records. 

Reviewer: ```suggestion
**Step 3:** Provide the Uni schema. Copy and paste the Uni [schema.json](uni_configuration/schema.json). The schema is the same as the one you used in Pattern 2. The difference is in the `aclnput` you used in the GraphQL query that you used to add new records. 
```

  
4. Press `Create`. Wait for 5 minutes for Share to finish Uni provisioning. 

> After Uni Provisioning is complete. You will see that your Uni will appear in the `RUNNING` status on the home page. In ‘My Nodes’ you will see two nodes: `CRM` and `Marketing`. 

Reviewer: ```suggestion
You will see that your Uni will appear in the `RUNNING` status after Uni Provisioning is complete. In ‘My Nodes’ you will see two nodes: **CRM** and **Marketing**. 
```

## Explore the Uni

> Let’s test how permission boundaries will work between the `CRM` and `Marketing` node. 

Reviewer: ```suggestion
Let’s test how permission boundaries will work between the **CRM** and **Marketing** nodes. 
```

> Using the `CRM` Node. Add the following mutation using `GraphQL Explorer`. 

Reviewer: ```suggestion
Add the following mutation using the `GraphQL Explorer` of the **CRM** node.. 
```

```graphql
mutation m {

  put_CustomerAccount_async(id: "", input: {address: {zipcode: 95678}, customerId: "1001", firstName: "Jay", lastName: "Pipes"}, 
aclInput: {acl: {principal: {nodes: "*"}, operations: READ}})

}
```

> Use `Marketing` node `Entity Explorer` to `Edit` the above `CustomerAccount` record and try to change the last name from `Pipes` to `Jaeger`. You will see an error. 

Reviewer: ```suggestion
Use the `Entity Explorer` on the **Marketing** node `Entity Explorer` to `Edit` the above `CustomerAccount` record. Try to change the last name from `Pipes` to `Jaeger`. You will see an error. 
```

> Use `Marketing` node `Entity Explorer` to `Edit` the above `CustomerAccount` record and try to change the last name from `Pipes` to `Jaeger`. You will see an error. 

Reviewer: What is the error?
